CHANGELOG_CLOUD.md

```diff
@@ -0,0 +1,685 @@
+# Changelog
+All notable changes to this project for *our cloud releases* will be documented in this file.
+The v23.0.0 release combines the open source and cloud releases and we won't have any
+separate cloud releases going forward.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+
+## [v23.0.0-beta1] - 2021-10-30
```

Comment:
typo in date

Reply:
below line add 
`[v23.0.0-beta1]: https://github.com/dgraph-io/dgraph/compare/v22.0.2...v23.0.0-beta1`

Reply:
Done

Reply:
still missing link

Reply:
The reason I am not adding the link is that this is not the correct diff.

Reply:
Adding this link https://github.com/dgraph-io/dgraph/compare/v21.03.0-96-g65fff46c4-slash...v23.0.0-beta1 but this link points to empty diff because of the way GitHub computes the diff.